src/stages/models/manager.ts

```diff
@@ -113,6 +120,13 @@ export class StageManager {
 
   @withSpanAsyncV4
   public async getStages(params: StageFindCriteriaArg): Promise<StageModel[]> {
+    const spanActive = trace.getActiveSpan();
```

Comment:
same as with jobs, attributes are not interesting

Reply:
done